### DISCUSSION TITLE
Prevent episodes in folders with the same AniDB ID from combining into one series

### DISCUSSION THREAD

**Discussion by doolydooly** [ZeroQI/Absolute-Series-Scanner#504] [Ideas]
I would like some way to prevent ASS from combining all episodes in folders with the same AniDB ID into the same series.

For me this has come up because One Piece is remastering a ~50 episode arc and condensing it into ~20 episodes. AniDB hasn't split these into a new series and has instead named with the O episode type, so ASS will group them into the Specials folder. This will put them into the Specials folder as Episode ~410-430 and will insert all of them into the overall play order between episodes 1122 and 1123. Since this is optional content, I want to be able to separate it from the main series (ideally into a separate series altogether), but still get metadata from AniDB for the episodes.

Essentially, I am just looking for a way to prevent ASS from merging episodes in folders with the same AniDB ID into the same series. Then I could place these episodes into a separate folder with the same AniDB ID as the main series and change the name/image/other series metadata manually as I see fit while still retaining the overall metadata (like cast & crew) from the main series.

I was thinking this could potentially be handled by appending something to the folder name, similarly to how season and episode offsets can be adjusted.

**Comment by ZeroQI** [comment]
ASS scanner will do season and episode offset and remove it from the forced id so it will be identical
If you did create separate folders, it will be assigned the same unique id and group them

You should have an option in Plex to split series for such cases...
